appveyor.yml

```diff
@@ -0,0 +1,28 @@
+image: ubuntu1804
+stack: node 8
+build: off
+
+cache:
+  - node_modules -> package.json
+
+install:
+  - sudo apt-get update
+  - sudo apt-get -fy install composer
+  - npm install -g gulp
```

Comment:
I have found it gives faster builds if you don't install gulp as a global, but instead use the one installed as a project dependency instead.
This cuts out a lot of double installing of everything Gulp.

To still allow building afterwards, add a scripts section to the package.json file and something like a "build" command to run gulp. This is because in there, you can use project-local commands as normal.
This is also nice because it standardizes a way to build your application.

Reply:
Agreed! I'll make a PR to Municipio and see what they think.

Reply:
@jofo8948 Gulp is no longer installed globally with e8cb336!

Builds are significantly faster now, thank you!